task/prepare.go

```diff
@@ -12,17 +12,15 @@ import (
 	"github.com/livepeer/stream-tester/segmenter"
 )
 
-var profile240p = livepeerAPI.Profile{
-	Name:    "240p0",
-	Fps:     0,
-	Bitrate: 250000,
-	Width:   426,
-	Height:  240,
-	Gop:     "2.0",
-}
-
 var allProfiles = []livepeerAPI.Profile{
-	profile240p,
+	{
+		Name:    "240p0",
+		Fps:     0,
+		Bitrate: 250000,
+		Width:   426,
+		Height:  240,
+		Gop:     "2.0",
```

Comment:
it's the length of the "group of pictures" https://en.wikipedia.org/wiki/Group_of_pictures

I'm not really sure what is the point of setting this though since we only do 1-to-1 segment transcoding. The GOP being set as 2 seconds might make 1 longer segment have... multiple keyframes? That could be interesting but I guess only if we actually broke up that output segment into multiple ones that can be served and buffered independently on the player, etc.

For now I've just copied this from the places where we create these profiles, like the API/dashboard code, the API client from `stream-tester`, etc. It also hasn't changed on this PR.